hist/hist/src/TH1.cxx

```diff
@@ -5005,13 +5014,14 @@ void TH1::GetBinXYZ(Int_t binglobal, Int_t &binx, Int_t &biny, Int_t &binz) cons
 /// is evaluated, normalized to one.
 ///
 /// @param rng (optional) Random number generator pointer used (default is gRandom)
+/// @param option (optional) Set it to "width" if your non-uniform bin contents represent a density rather than counts
 ///
 /// The integral is automatically recomputed if the number of entries
 /// is not the same then when the integral was computed.
-/// NB Only valid for 1-d histograms. Use GetRandom2 or 3 otherwise.
+/// @note Only valid for 1-d histograms. Use GetRandom2 or GetRandom3 otherwise.
 /// If the histogram has a bin with negative content a NaN is returned
```

Comment:
```suggestion
/// @note Only valid for 1-d histograms. Use GetRandom2 or GetRandom3 otherwise.
/// If the histogram has a bin with negative content, a NaN is returned.
```